mage_ai/data_cleaner/column_type_detector.py

```diff
@@ -54,143 +57,114 @@ def get_mismatched_row_count(series, column_type):
     mismatched_rows = 0
     if column_type == EMAIL:
         mismatched_rows = len(
-            series[~series.str.contains(REGEX_EMAIL)].index,
+            series[~series.str.match(REGEX_EMAIL)].index,
         )
     elif column_type == PHONE_NUMBER:
         mismatched_rows = len(
-            series[~series.str.contains(REGEX_PHONE_NUMBER)].index,
+            series[~series.str.match(REGEX_PHONE_NUMBER)].index,
         )
     elif column_type == ZIP_CODE:
         str_series = series.astype(str)
         mismatched_rows = len(
-            series[~str_series.str.contains(REGEX_ZIP_CODE)].index,
+            series[~str_series.str.match(REGEX_ZIP_CODE)].index,
         )
     return mismatched_rows
 
+def clean_whitespace(series):
```

Comment:
Is this method used in multiple places?

Reply:
It is not used in multiple places, this function is now inlined.